jest/private/jest_test.bzl

```diff
@@ -65,7 +65,10 @@ def _impl(ctx):
     if ctx.attr.chdir:
         unwind_chdir_prefix = "/".join([".."] * len(ctx.attr.chdir.split("/"))) + "/"
 
-    fixed_args = [
+    fixed_args = []
+    if hasattr(ctx.attr, "fixed_args"):
```

Comment:
Is the `hasattr` only required because an older version of `rules_js` might be used?

Reply:
Yea, shouldn't happen with bzlmod but it can with someone's poorly written WORKSPACE file.

Reply:
Can we drop it at some point?

Reply:
For rules_jest we can with the 1.0 release since its still pre-1.0. I'll add a TODO.

Reply:
Yeah I'd like to add TODOs to all these so we remember to drop it in the future.